src/lib/modules/response/resp_delay.go

```diff
@@ -22,9 +21,11 @@ const (
 // in a Route struct and supplies it to a function in turn to handle it accordingly. We create
 // channels to run tests for each route in parallel, speeding up the process
 func HandleResponseDelayForRoute(responseChains []*tsdb.Chain, route parser.Routes, tsdbNameHash string, wg *sync.WaitGroup) {
-	routeSuffix := filters.RouteDestroyer(route.URL + "_" + route.Route)
+	// routeSuffix := filters.RouteDestroyer(route.URL + "_" + route.Route)
+
 	// Init paths for request-response-monitoring
-	path := PathReqResDelay + "/" + "chunk_req_res_" + routeSuffix + ".json"
+	path := PathReqResDelay + "/" + "chunk_req_res_" + tsdbNameHash[12:len(tsdbNameHash)] + ".json"
```

Comment:
I just did this to make it work, this is not the best way, please restructure it, check how it works with other modules.